root/static/scripts/edit/URLCleanup.js

```diff
@@ -2337,6 +2337,34 @@ const CLEANUPS: CleanupEntries = {
       return url;
     },
   },
+  'jazzmusicarchives': {
+    match: [new RegExp('^(https?://)?(www\\.)?jazzmusicarchives\\.com', 'i')],
+    restrict: [LINK_TYPES.otherdatabases],
+    clean: function (url) {
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?jazzmusicarchives\.com\/([^#]+)(?:#.*)?$/, 'https://www.jazzmusicarchives.com/$1');
+      return url;
+    },
+    validate: function (url, id) {
+      const m = /^https:\/\/www\.jazzmusicarchives\.com\/(\w+)\/(?:[\w-]+\/)?[\w-]*$/.exec(url);
+      if (m) {
+        const type = m[1];
+        switch (id) {
+          case LINK_TYPES.otherdatabases.artist:
+            return {
+              result: type === 'artist',
+              target: ERROR_TARGETS.ENTITY,
+            };
+          case LINK_TYPES.otherdatabases.release_group:
+            return {
+              result: type === 'album',
+              target: ERROR_TARGETS.ENTITY,
+            };
+        }
+        return {result: false, target: ERROR_TARGETS.ENTITY};
```

Comment:
The Metal Archives are allowed to link to labels and releases. Should Jazz Music Archives be the same?

Reply:
Jazz Music Archives does not seem to have label pages, so those are out. 

http://www.jazzmusicarchives.com/album/john-coltrane/my-favorite-things (for example) does say "Atlantic 1361(US)" which makes it seem like these map to releases, but http://www.jazzmusicarchives.com/album/miles-davis/kind-of-blue for example says "Sony CD Reissue in 1997 includes 1 bonus track" and nothing else - they don't seem to have an entry for said reissues. So I think RG is probably the most reasonable choice to map to.

Reply:
Ok thanks, just asking because it is the same network of websites.

Reply:
Actually, I think the original user was a bit confused - the site in the same network is http://www.metalmusicarchives.com/ as they linked, but we do not have that - we have https://www.metal-archives.com/ which is a completely different one :) People are just not very original.